### ISSUE TITLE
Accessibility - Video Player - Screen reader & Keyboard - Slider control for Forward /rewind and volume control no label.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/ibm-dotcom-library#1772]
## Detailed description

Issue - (Screen reader )
Slider control for Forward /rewind and volume control doesn't have a Label 

Expected - Provide a descriptive label which conveys the purpose of the control

Issue - (Keyboard)
There are no separate slider controls for Volume and video forward and rewind
In the same slider, the user can use Up/Down arrow to control the volume. And Left/Right Arrow keys to forward /Rewind the video. But, the user is not informed about this behaviour in advance.

Expected -
1. Either provide separate controls for adjusting volume and Video forward and rewind
Or,
2. Inform the user how to interact in the slider

**Comment by RobertaJHahn** [comment]
Moving this issue to sprint 20-08.  The dev work has to be done during the first week of the sprint to include it in release v1.6.0.

**Comment by RaphaelAmadeu** [comment]
This issue is happening on the embed player, so we can't fix this on our end.
This issue can be closed too

CC: @jeffchew @RobertaJHahn @ljcarot

**Comment by ljcarot** [comment]
Closing this issue.  Will send details to video team to discuss with the Kaltura product owner.
